examples/browser/test/pages/main-page.ts

```diff
@@ -36,19 +38,23 @@ export class MainPage {
         }
     }
 
+    /* returns true if the menu and the main working area in Theia are loaded */
     public waitForLoadingPanels(): void {
         this.driver.waitForExist('#theia-top-panel');
         this.driver.waitForExist('#theia-main-content-panel');
     }
 
+    /* Returns true if the Main work area in Theia is loaded */
     public isMainContentPanelLoaded(): Boolean {
-        if (this.driver.element('#theia-main-content-panel').state === 'success') {
+        if (browser.element('#theia-main-content-panel').isExisting()) {
```

Comment:
Why do not you just write `return browser.element('#theia-main-content-panel').isExisting();`?